src/instance.rs

```diff
@@ -214,6 +225,123 @@ unsafe impl<T> AsPyPointer for Py2<'_, T> {
     }
 }
 
+/// A borrowed equivalent to `Py2`.
```

Comment:
I think we should just give up and call this `Py3`. ;-) But more seriously, I think we should get rid off `Py2` sooner or later and commit to some reasonable. By now the pain of reading `Py2` has made my mind pliable and I will probably accept almost anything even remotely sane.

Reply:
Heh, maybe I will open a PR to rotate `Py<T> -> PyDetached<T>` (or `PySend<T>`) and `Py2<'py, T> -> Py<'py, T>`. 🙈